airdrop/constants.py

```diff
@@ -16,3 +16,7 @@ class AirdropClaimStatus(Enum):
 class AirdropEvents(Enum):
     AIRDROP_CLAIM = 'Claim'
     AIRDROP_WINDOW_OPEN = 'AirdropWindowOpen'
+
+
+class TokenName(Enum):
```

Comment:
Why do we need an Enum here? We are going to have only one token and it needs to be part of config.